docs/end-user-guide/using-the-plugin.md

```diff
@@ -0,0 +1,11 @@
+## Connecting an account
+
+Use the `/connect` slash command to connect.
```

Comment:
@larkox, is the user connecting their account in Mattermost or connecting a Teams account to Mattermost?

Reply:
The user is, let's say "linking" their Teams account to Mattermost.

When they run `/mstmeetings connect` (I think that is the right name of the command), they will be prompted to log in into Microsoft Teams.

Reply:
@justinegeffen this is not yet updated, right?